container/shim/src/fetchers/lassie.js

```diff
@@ -160,13 +160,17 @@ function createLassieURL(req, isRawFormat) {
     // translate depth parameter for lassie
     let newKey = key;
     let newVal = val;
-    if (key === "depth" && (val === "1" || val === "0")) {
-      newKey = "depthType";
-      newVal = "shallow";
+    if (key === "depth" && val === "1") {
+      newKey = "car-scope";
+      newVal = "file";
+    }
+    if (key === "depth" && val === "0") {
+      newKey = "car-scope";
+      newVal = "root";
```

Comment:
@hannahhoward is this a typo or did we implement different things? 

https://github.com/ipfs/bifrost-gateway/issues/80 → https://github.com/ipfs/bifrost-gateway/pull/81/files → I did `block` instead of `root` as we used both durign discussion, and `block` felt more descriptive  :see_no_evil:   

(not the end of the world, no need to revert releease, bifrost-gateway still sends  `depth=0`  so it will work, but we should clean this up before we remove `depth`)